workspaces/mend/plugins/mend/src/api/index.ts

```diff
@@ -0,0 +1,153 @@
+import { QueryClient } from '@tanstack/react-query';
+import { createApiRef, DiscoveryApi } from '@backstage/core-plugin-api';
+import { CatalogApi } from '@backstage/catalog-client';
+
+/**
+ * Plugin API
+ */
+export type MendApi = {
+  discoveryApi: DiscoveryApi;
+};
+
+export const mendApiRef = createApiRef<MendApi>({
+  id: 'plugin.mend.service',
+});
+
+export class MendClient {
+  discoveryApi: DiscoveryApi;
+
+  constructor(options: { discoveryApi: DiscoveryApi }) {
+    this.discoveryApi = options.discoveryApi;
+  }
+}
+
+/**
+ * Catalog API Ref
+ */
+export const catalogApiRef = createApiRef<CatalogApi>({
+  id: 'plugin.catalog.service',
```

Comment:
I'm not sure why this is needed, you should be able to import the existing `catalogApiRef`

Reply:
Best I can tell this actually isn't used anywhere